outputs/influxdb/influxdb.go

```diff
@@ -49,6 +50,10 @@ var sampleConfig = `
   # user_agent = "telegraf"
   # Set UDP payload size, defaults to InfluxDB UDP Client default (512 bytes)
   # udp_payload = 512
+
+  # Route metrics to an InfluxDB database based on value of a tag
+  # note: if this tag has many unique values, write performance may suffer
```

Comment:
Not sure about this note, is it just because there will be multiple batch points? I think that using this option in general will negatively affect performance